share/completions/blender.fish

```diff
@@ -0,0 +1,125 @@
+function __blender_list_scenes -a file
+    stdbuf --output=0 blender --background test.blend --python-expr 'import bpy
```

Comment:
stdbuf is non-standard - it's a GNU thing, we can't use it unless absolutely necessary.

Reply:
Without it piping to head/tail messes up the stdout output:

![image](https://github.com/fish-shell/fish-shell/assets/42812113/1195a782-a605-46f6-9df8-f5131c779b4c)

Reply:
See above, you want to match the lines you're interested in.

Also: What `stdbuf` is doing here is turning off buffering on stdout. If that changes the order things appear in, that's suspect.

I would double-check that things aren't being printed on stderr (which is unbuffered by default!).

Reply:
I've added a guard condition to print nothing when stdbuf is not available. I've tried redirect `stderr` to `/dev/null` but it didn't help, unfortunately.